test/wat_i.t

```diff
@@ -0,0 +1,335 @@
+  $ run_it() { PACMAN_LOG="$TESTDIR"/pacman.log "$TESTDIR"/../wat "$@"; }
```

Comment:
Alrighty: I wasn't sure how extensive or not it ought to be. I've very rarely ever dealt with automated code tests. I'll merge it back into wat.t